Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -55,9 +53,47 @@ export type StatusBarAnimation = $Enum<{
   slide: string,
 }>;
 
-type DefaultProps = {
-  animated: boolean,
-};
+type StatusBarProps = $ReadOnly<{|
+  /**
+   * If the status bar is hidden.
+   */
+  hidden?: ?boolean,
+  /**
+   * If the transition between status bar property changes should be animated.
+   * Supported for backgroundColor, barStyle and hidden.
+   */
+  animated?: ?boolean,
+  /**
+   * The background color of the status bar.
+   * @platform android
+   */
+  backgroundColor?: ?string,
```

Comment:
When there are a bunch of Props that are supported on different platforms, I like doing this pattern:

```
type AndroidProps = //
type IOSProps = //

type Props = $ReadOnly<{|
  ...IOSProps,
  ...AndroidProps,
  // shared props
|}>
```

Reply:
And also re-organized these